Framework/DataHandling/src/LoadPSIMuonBin.cpp

```diff
@@ -552,8 +589,17 @@ void LoadPSIMuonBin::assignOutputWorkspaceParticulars(
   for (auto i = 0u; i < sizeOfLabels; ++i) {
     if (m_header.labelsOfHistograms[i] == "")
       break;
-    addToSampleLog("Label Spectra " + std::to_string(i),
-                   m_header.labelsOfHistograms[i], outputWorkspace);
+    std::string name = m_header.labelsOfHistograms[i];
+    // if no name is present (or just empty space)
```

Comment:
If no name is present we break above?

Reply:
if the name is an empty string we break. If it is a string with no content (i.e. just blank space) we need to do things differently. I have made the comment more clear.